bloom-housing/bloom
# fix: allow a due date for open waitlist

This PR addresses #4231 


## Description

Adds additional condition to display `Is there an application due date?` question when open waitlist selected.
Before as a workaround we could just switch to `available units` set due date, and switch to `open waitlist`. Now it can be added from open waitlist

## How Can This Be Tested/Reviewed?

On partners switch `What is the listing availability?` to open waitlist. Now in `Rankings & Results` we should see `Is there an application due date?` question

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
